app/controllers/accounts_controller.rb

```diff
@@ -39,6 +40,19 @@ def archive
 
   private
 
+  def attach_avatar
+    account.avatar.attach(attaching_image) if attaching_image
+  end
+
+  def attaching_image
+    image_params = params.fetch(:cropped_image, params.dig(:account, :cropped_image))
+    return unless image_params.present?
+
+    { io: StringIO.new(Base64.decode64(image_params.split(',')[1])),
```

Comment:
Is there a solution that's not based on base64? Can we use multipart form instead? Why not?